core/services/keystore/keys/ocr2key/export.go

```diff
@@ -33,22 +34,23 @@ func FromEncryptedJSON(keyJSON []byte, password string) (KeyBundle, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to decrypt OCR key")
 	}
+	var kb KeyBundle
 	switch export.ChainType {
 	case chaintype.EVM:
-		key := mustNewKeyFromRaw[*evmKeyring](rawKey, &evmKeyring{})
-		return &key, nil
+		kb = emptyKeyBundle(new(evmKeyring))
 	case chaintype.Solana:
-		key := mustNewKeyFromRaw[*solanaKeyring](rawKey, &solanaKeyring{})
-		return &key, nil
+		kb = emptyKeyBundle(new(solanaKeyring))
 	case chaintype.Terra:
-		key := mustNewKeyFromRaw[*terraKeyring](rawKey, &terraKeyring{})
-		return &key, nil
+		kb = emptyKeyBundle(new(terraKeyring))
 	case chaintype.Starknet:
-		key := mustNewKeyFromRaw[*starknetKeyring](rawKey, &starknetKeyring{})
-		return &key, nil
+		kb = emptyKeyBundle(new(starknetKeyring))
 	default:
 		return nil, chaintype.NewErrInvalidChainType(export.ChainType)
 	}
+	if err := kb.Unmarshal(rawKey); err != nil {
+		panic(err)
```

Comment:
Better to return the error now that we aren't calling must directly?

Reply:
I usually favor letting the caller decide when the signature already supports `error` - not much to update in this case either.